contracts/Migrations.sol

```diff
@@ -1,4 +1,4 @@
-pragma solidity ^0.4.17; 
+pragma solidity ^0.4.17;
```

Comment:
Oh sure, I just didn't want to mess around with your versioning. However, wouldn't this be easier if you did it on your end instead of waiting for travis run ci again?

Reply:
I think it makes to do here since we are already changing the Migrations files. All other sol files have been updated to ^0.4.24 already. The Travis CI should be pretty quick. We'll get this merged in right after

Reply:
Okay, here we go!